### ISSUE TITLE
knowledge base error

### ISSUE THREAD

**Issue by 486DX2** [crypto2011/IDR#82]
Hi

When trying to open a delphi EXE, I get this error:

![afbeelding](https://user-images.githubusercontent.com/120501674/208965453-4c402b08-5320-4540-a63f-b960dd8647f4.png)

**Comment by dartraiden** [comment]
Read the readme and download knowledge bases.

**Comment by 486DX2** [comment]
I have to extract all 7z files too, right?
Because after downloading it and putting it there, it still gives this error.

**Comment by 486DX2** [comment]
Can you explain me what a knowledge base is and what's inside those files?
I'm a complete beginner

**Comment by dartraiden** [comment]
Yes, you need to extract all 7z and put extracted files beside IDR.exe

**Comment by greenozon** [comment]
@ghost   were you able to solve your issue?
